gulpfile.js

```diff
@@ -22,9 +22,11 @@ gulp.task('assets:app', function() {
 
 gulp.task('assets:vendor', function() {
   return gulp.src([
-    './node_modules/budgetkey-ng2-components/assets/**/*'
+    // './node_modules/budgetkey-ng2-components/assets/**/*'
+    './node_modules/openprocurement-ng2-components/assets/**/*'
```

Comment:
We should aim for the theme selection to be applied in one of two ways:
- Environment variable (i.e. same docker image, theme is selected at run time according to the value of an env var)
- Docker image inheritance (i.e. you create a new Docker image in which you only overwrite a single `theme.json` file which contains the theme specification.

Changing dependencies and requiring to rebuild the entire thing is not flexible enough and will force us to maintain two codebases which is something to avoid.
Also see my comment here https://github.com/OpenBudget/budgetkey-ng2-components/pull/3#issuecomment-335370366